runtime-modules/storage/src/tests/data_object_storage_registry.rs

```diff
@@ -155,3 +157,33 @@ fn test_toggle_ready() {
         );
     });
 }
+
+#[test]
```

Comment:
I think these tests will also move into the working-group module

Reply:
Agree.